admin/data/sql/migrations/077.do.sql-session-store.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE [mtc_admin].[sessions](
+  [sid] varchar(255) NOT NULL
+    CONSTRAINT [pk_sessions] PRIMARY KEY CLUSTERED ([sid] ASC),
+  [expires] datetimeoffset NOT NULL,
```

Comment:
Do we want to index the `expires` field - then we can run a query based on this col.

Reply:
given the overhead of indexing millisecond precision dates i would rather not.  The situations where we would need to query this are minimal, and would likely be done offline.

if it were a common lookup we could look at that.